crates/sui-network/src/state_sync/mod.rs

```diff
@@ -407,14 +414,21 @@ where
                 .sequence_number(),
         );
 
+        // Spawn tokio task to update metrics periodically in the background
+        let (_sender, receiver) = oneshot::channel();
+        tokio::spawn(update_highest_sync_metrics(
```

Comment:
remove 402-409 as well?

Reply:
sure, i will remove the above metrics update. I think the bg task will update them anyways